Gemfile

```diff
@@ -142,6 +142,6 @@ gem 'blacklight_range_limit', '6.5.0'
 gem 'dog_biscuits', git: 'https://github.com/samvera-labs/dog_biscuits.git'
 gem 'hyrax-v2_graph_indexer', git: 'https://github.com/scientist-softserv/hyrax-v2_graph_indexer', branch: 'main'
 # rubocop:disable Metrics/LineLength
-gem 'iiif_print', git: 'https://github.com/scientist-softserv/iiif_print.git', ref: '56069b5f8bbcbf6629b9973352069d21d5ddaf0f'
+gem 'iiif_print', git: 'https://github.com/scientist-softserv/iiif_print.git', branch: 'add-models-to-pending-relationships-table'
```

Comment:
TODO: `bundle update iiif_print --conservative` once the corresponding PR gets approved/merged into iiif_print.